taca/cleanup/cleanup.py

```diff
@@ -211,10 +211,20 @@ def cleanup_irma(days_fastq, days_analysis, only_fastq, only_analysis, status_db
     # make a connection for project db #
     pcon = statusdb.ProjectSummaryConnection(conf=status_db_config)
     assert pcon, "Could not connect to project database in StatusDB"
+    
+    # make exclude project list if provided
+    exclude_list = []
+    if exclude_projects:
```

Comment:
I would suggest to make a sanity check on the projects, in order to avoid failures in the next steps.. it is true that you check for the presence of projects in this list but there might be nasty cases. A possible wa is to check if all elements added to`exclude_list` point to existing folders.